### DISCUSSION TITLE
Updating / Upgrade from DKAN 1 to  DKAN 2

### DISCUSSION THREAD

**Discussion by BFriddy** [GetDKAN/dkan#3843] [Migration help]
Sorry if this has already been addressed / answered. But I was wondering if it's possible to upgrade / migrate from DKAN ver. 1 to ver.2? If so, can someone point me in the right direction for the documentation around this?

Thanks

**Comment by janette** [comment]
@BFriddy depending on the complexity of your DKAN v1 site, the easiest path would be to stand up a new v2 site, and harvest the data from your v1 site. However, if you need to retain the users, authorship, revisions, etc. then it will be a bit more trial and error. Here are a few helper modules but not much documentation. 

- https://github.com/GetDKAN/dkan_export
- https://github.com/GetDKAN/dkanclassic_import
